pom.xml

```diff
@@ -520,6 +520,30 @@
                     </ignoredResourcePatterns>
                 </configuration>
             </plugin>
+             <plugin>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <version>3.0.0-M2</version>
+        <executions>
+          <execution>
+            <id>enforce-maven</id>
+            <goals>
+              <goal>enforce</goal>
+            </goals>
+            <configuration>
+              <rules>
+                <requireMavenVersion>
+                  <version>3.5.2</version>
```

Comment:
@gitcliff , since this plugin is already added Transitively from the [hapi-fhir](https://github.com/hapifhir/hapi-fhir/blob/master/pom.xml#L2235-L2261)  dependence, you do not have to add it again in oder to overide it ,
you can just create a simple profile to overide the <requireMavenVersion>  in the TRavis enviroment.
you could name the Profile `travis`  and then run the profile in  [travis](https://github.com/openmrs/openmrs-contrib-plm-fhir-server/blob/master/.travis.yml#L32) with   `- mvn -B install -P travis`